soundbay/conf/augmentations/_augmentations.yaml

```diff
@@ -1,20 +1,24 @@
 # @package _global_
 _augmentations:
   random_noise:
-    _target_: soundbay.data_augmentation.AddGaussianNoise
+    _target_: audiomentations.AddGaussianSNR
```

Comment:
Did you run an experiment to see how these audiomentations work compared to our "self built" methods?

Reply:
Not yet, my next task is compiling a set of augs I belive can do better than our current augs. This is just an infrastructure change.